proto/panacea/datapool/v2/pool.proto

```diff
@@ -60,9 +61,11 @@ message PoolParams {
   cosmos.base.v1beta1.Coin nft_price = 4;               // A price of a NFT
   repeated string          trusted_data_validators = 5; // Trusted data validators' addresses
   repeated string          trusted_data_issuers = 6;    // Optional. Data issuers that is trusted by the curator
-  google.protobuf.Duration download_period = 7 [
```

Comment:
With removing the `download_period`, we don't have to import "google/protobuf/duration.proto" in this .proto file

Reply:
Yes. I fixed it.

https://github.com/medibloc/panacea-core/pull/320/commits/9a32cf6431630405f85e9ceaf84c83c72b990862